packages/blade/rollup.config.mjs

```diff
@@ -12,7 +13,10 @@ import pluginAlias from '@rollup/plugin-alias';
 import pluginReplace from '@rollup/plugin-replace';
 import ts from 'typescript';
 import { depsExternalPlugin } from './dependencies-external-plugin.mjs';
-import packagejson from './package.json' assert { type: 'json' };
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const packagejson = JSON.parse(fs.readFileSync(path.resolve(__dirname, './package.json'), 'utf8'));
```

Comment:
`assert` was changed to `with` in node 22

`assert` breaks in node 22, `with` breaks in node 20 🙈  so changed it to good old fs.readFile